users/views.py

```diff
@@ -36,4 +37,27 @@ def post(self, request):
         except KeyError:
             return JsonResponse({'message' : 'KEY_ERROR'}, status=400)
 
+class SigninView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            user = User.objects.filter(email = data['email'])
+
+            if not bcrypt.checkpw(data['password'].encode('utf-8'), user.password.encode('utf-8')):
+                raise ValidationError('INVALID_USER')
```

Comment:
굳이 error를 raise할 필요가 없어보입니다.
```suggestion
                return JsonResponse({'message' :  'INVALID_PASSWORD'}, status=401)
```

Reply:
확인했습니다